### PR TITLE
TC_00.002.24 | Creating new item > Creating Pipeline

### PR DESCRIPTION
**Pull request by goryanka**
https://trello.com/c/cgiRvhCG/620-tc0000224-creating-new-item-creating-pipeline

### CODE REVIEW

**Review by AliaksandrMakaranka — APPROVED** [review]

File: src/test/java/school/redrover/Pipeline15Test.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+public class Pipeline15Test extends BaseTest {
+    private void createApiPipeline(String jobName) {
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/Pipeline15Test.java:9]: I would add an empty line on 9 and change a name of a method (it looks like something connected with API)

File: src/test/java/school/redrover/Pipeline15Test.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public void testCreateJob() {
+        final String jobName = "Pipeline_1";
+
+        createApiPipeline(jobName);
+
+        goToDashBoard();
+
+        String actual = getDriver().findElement(By.xpath("//a[@class='jenkins-table__link model-link inside']")).getText();
+        Assert.assertEquals(actual, jobName);
+
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/Pipeline15Test.java:34]: I would delete an empty line on 34.

**Review by VictoriaLema — APPROVED** [review]

File: src/test/java/school/redrover/Pipeline15Test.java (hunk 2 above)
Comment (goryanka) [reply src/test/java/school/redrover/Pipeline15Test.java]: done

File: src/test/java/school/redrover/Pipeline15Test.java (hunk 1 above)
Comment (goryanka) [reply src/test/java/school/redrover/Pipeline15Test.java]: done

**Pull request merged**